docs/components/search/test.html

```diff
@@ -328,4 +328,37 @@ <h3>Required</h3>
       </table>
     </section>
   </section>
+
+  <section>
```

Comment:
Added test for an extra long value.

Reply:
Can you add two more tests?

* Long + .hxInvalid
* Long + [disabled]

Reply:
Added the two additional tests:

* Long + .hxInvalid
* Long + [disabled]